### ISSUE TITLE
Feedback for StudyGroup (early days)

### ISSUE THREAD

**Issue by murraycadzow** [smilefreak/studyGroup#21]
We would love people to give us some feedback on how our study group is going
Don't limit it to just these but here are some questions
Is the content too hard, too easy?
Are the topics useful?
How could we improve what is being presented?
What would you like to learn from these sessions?

We'd really like to get a discussion going on what is going well, and what could be improved

**Comment by Amara-Shaukat** [comment]
a) Sometimes I find it easy but by understanding quite new thing, program or software I get into trouble.
b) Yup, contents are useful
c) Present new things with level 1 or low level so it could be understandable. Then make it fast as we get familiar with it. Make contribution of other participants like they can represent their understanding about programming and how they are using it on their stuff.
d) I want to get skills regarding programing so I could deal my stuff efficiently and productively.

**Comment by lastkarrde** [comment]
I'd like to attend but have work until 5pm. Interested to see if any others are in the same boat?

**Comment by murraycadzow** [comment]
Thanks Amara and Logan for the feedback.

I think re time we'll stick to the current 15:30 until the end of the year and reassess next year since peoples schedules will be different

It would be great to build on this so we can get a good idea on where it is best to pitch the level for the group and then how to also ensure that we have a way to enable people that find the content too hard or the pace too fast to still follow along and develop their skills too.

Feedback like this will help us gauge what level and speed the group would like

**Comment by Geeketics** [comment]
When I'm there it seems real useful, but as soon as I get back to my desk I don't try to use the new things I just learnt because it'd take too long to work out - but then later once I'm not so busy perhaps I would be more willing to try things. Hard to judge.

I liked James' challenge in the Python stuff, but didn't have time to try during the rest of the week so perhaps making full challenge style sessions or half sessions would be cool.

Maybe showing an example of what the person presenting uses whatever they're teaching would make it more relatable too, rather than just this is how, get some this is why going on.

**Comment by methylnick** [comment]
great SYSKA yesterday, happy to facilitate an [Authroea](http://www.authorea.com) SYSKA or session if this is of interest to anyone, this would add onto Riki's Latex one. In terms of Bioconductor, there is something that would be of interest to you all [Bioconductor RNA-Seq in Jupiter](http://www.rna-seqblog.com/bioconductors-rna-seq-workflow-in-jupyter-notebook-format/?utm_source=RNA-Seq+Blog+Subscribers&utm_campaign=fe2b82265a-RSS_EMAIL_CAMPAIGN&utm_medium=email&utm_term=0_e9607034ee-fe2b82265a-85920497)

Cheers Nick.

**Comment by Amara-Shaukat** [comment]
Yesterday session was quite informative especially in terms of learning Latex and Bioconductor. Thanks Murray, Ritu and Mick. Hopefully, future sessions would be more useful.

**Comment by anna-gosling** [comment]
Yeah, thanks everyone. The wee five minute sessions were a good way of introducing new tools. I look forward to the LaTeX session in future (thanks Riku - if I had more than a week before I wanted to submit, I'd definitely be giving it a go right now - battling with formatting in Word at the moment. It really does not enjoy thesis length documents. Much swearing going on. It's really not pretty).

**Comment by smilefreak** [comment]
Alright I am going to close this now. There is some good feedback in here, sounds like yesterday was a blast.

 I will add a new label feedback so that each  session can have it's own feedback, this will be especially useful for each presenter after individual sessions. If you have a particular topic of feedback you would like to discuss just open an issue and we can discuss there. Thanks heaps.

**Comment by murraycadzow** [comment]
I enjoyed the first round of syska's yesterday, I think what I learned most was just how little time 5 minutes is and if we're to keep to that length, the presentation is really a very brief punchy summary of the topic

**Comment by methylnick** [comment]
yes, and for those studying postgrad, it is a great exercise to do elevator pitches, which is essentially what the SYSKA is, you have 5 mins to pitch your project to your next employer, what are the highlights, what must they know before they get out of the lift?
